src/control/server/instance.go

```diff
@@ -287,7 +287,7 @@ func (ei *EngineInstance) SetupRank(ctx context.Context, rank ranklist.Rank, map
 }
 
 func (ei *EngineInstance) callSetRank(ctx context.Context, rank ranklist.Rank, map_version uint32) error {
-	dresp, err := ei.CallDrpc(ctx, drpc.MethodSetRank, &mgmtpb.SetRankReq{Rank: rank.Uint32(), MapVersion: map_version})
+	dresp, err := ei.callDrpc(ctx, drpc.MethodSetRank, &mgmtpb.SetRankReq{Rank: rank.Uint32(), MapVersion: map_version})
```

Comment:
Please could you confirm that we are sure that the engine is ready when this function is invoked.

Reply:
callsetRank() is part of the (control-plane<->engine) setup routine that runs before the ready flag is set. dRPC connection should always be up when this is called.

Reply:
Thanks for the explanation :)